src/components/LicensePicker/__tests__/LicensePickerUtils.test.unit.js

```diff
@@ -23,9 +23,9 @@ describe('LicensePickerUtils', () => {
       'MIT-0': true,
       'MIT AND Apache-2.0': true,
       'MIT OR Apache-2.0': true,
-      junk: false,
+      junk: true,
```

Comment:
should the string `junk` be considered valid?

is it when it is invalid that is crashes the control?

Reply:
This is a change introduced by the usage of the custom Spdx validation lib.
Looks like that if any kind of wrong license is passed, then it will considered as valid since it would be changed in `{noassertion: null}`

Reply:
i see. makes sense